ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -983,6 +998,7 @@ spec:
         - input: "$(tasks.detect-changes.results.affected_catalogs)"
           operator: in
           values: [""]
+        - *bundleAdded
```

Comment:
The task already skips a upload process inside of the task. We don't have to skip it here.